config.py

```diff
@@ -61,10 +66,15 @@ class Test(Config):
     WTF_CSRF_ENABLED = False
     SERVER_NAME = 'localhost'
 
+    FEATURE_FLAGS_EDIT_SERVICE_PAGE = enabled_since('2015-06-03')
```

Comment:
Should this be False until we are ready to turn on editing services? i.e. once we have monitoring of changes in the admin app sorted out?